FreeRTOS/FreeRTOS
# Integration/timers

Description
-----------
For writing Unity tests that need full kernel source  -- no mocks -- on the Linux port.

Test Steps
-----------
Observe output of:
```
cmake -S . -B build 
cmake --build build
./build/timers/Timers
```

Related Issue
-----------
n/a


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
